_plotly_utils/basevalidators.py

```diff
@@ -43,12 +43,16 @@ def fullmatch(regex, string, flags=0):
 # Utility functions
 # -----------------
 def to_scalar_or_list(v):
+    if np and np.isscalar(v) and hasattr(v, 'item'):
+        return np.asscalar(v)
```

Comment:
I'm not sure I understand this branch.  What kinds of objects does this handle?

Reply:
I'll add a comment to the code.

Reply:
Thanks!